docs/guides/writing_stubs.rst

```diff
@@ -662,6 +717,14 @@ No::
 
     class OtherClass: ...
 
+Use variable annotations instead of type comments, even for stubs that target
+older versions of Python.
+
+Platform-dependent APIs
+-----------------------
+
+Use platform checks like `if sys.platform == 'win32'` to denote platform-dependent APIs.
```

Comment:
```suggestion
Use platform checks like ``if sys.platform == 'win32'`` to denote platform-dependent APIs.
```
I think we have a spec section that describes the allowed syntax for these; we could link to it here.